docs/scenario-dashboard.md

```diff
@@ -0,0 +1,244 @@
+  </div>
+
+#### Sensor Editor
+
+The Sensor editor tool enables the user to import a JSON sensor configuration file or build a sensor 
+configuration by adding and moving sensors on the vehicle model. Users are able to alter the 
+position and location for every sensors. Many sensors offer further alterations depending on the 
+sensor type such as image height, image width, and field of view. The "Import Config or Vehicle" 
+Button enables users to import a vehicle model and also can import a sensor json file for easy 
+uploading. After making all selections, the user needs to "Save Config" in the top left corner to 
+save the configuration.
+
+<div class="img_container">
+  <video width=650px height=480px muted autoplay loop>
+    <source src="https://cdn.monodrive.io/readthedocs/dashboard_sensor_editor.mp4" type="video/mp4">
+  </video>
+</div> 
+
+### Unit Under Test (UUT) & Scenarios
+
+ In the monoDrive Batch Scenario Manager, users will be able to run and configure a Unit Under Test. 
+ The Unit Under Test will be configured for closed loop or replay mode. monoDrive has many pre-built 
+ trajectory files to run using the "Straight 5K" Map.
+
+  <div class="img_container">
+    <img class='wide_img' src="../imgs/dashboard_scenarios.png"/>
+  </div>
+
+For closed loop mode, the user can add as many scenario JSON files to run in the batch. 
+
+  <div class="img_container">
+    <img class='wide_img' src="../imgs/dashboard_closed_loop.png"/>
+  </div>
+
+For details on writing a Unit Under Test and/or work with examples, see 
+[Configuring UUT](/scenario-dashboard/#connecting-your-code).
+
+### Weather
+
+The monoDrive Batch Scenario Manager offers many pre-build weather profiles and an option for 
+customizing a new weather profile. The weather configuration provides customizable changes such as 
+the angle of the sun, the amount of rain on the roads, amount of wind, wind angle, and rain fall 
+in a scene. 
+
+  <div class="img_container">
+    <img class='wide_img' src="../imgs/dashboard_weather.png"/>
+  </div>
+
+  <div class="img_container">
+    <img class='wide_img' src="../imgs/dashboard_custom_weather.png"/>
+  </div>
+
+## Running
+
+  1. Inside the downloaded folder for monoDrive Dashboard, double-click or open 
+  monodrive.dashboard.exe on Windows or monodrive-dashboard on Linux.
+
+  1. Open the Simulator or Scenario Editor.
+
+    <div class="img_container">
+      <img class='wide_img' src="../imgs/dashboard_simulator.png"/>
+    </div>
+ 
+  1. Run a [Unit Under Test](/scenario-dashboard/#connecting-your-code).
+
+  1. Choose or create a directory to save configuration files and results from the Unit Under Test 
+  and the Batch Scenario Manager. In the Unit Under Test Example provided, the user would send the 
+  same asset directory as set in the Batch Scenario Manager.
+
+  ``` python closed_loop_aebs.py --md_assets C:\Users\developer\Documents\BatchExample --md_loop ```
```

Comment:
block code is usually split onto multiple lines. not sure if this is what's causing the formatting issue

e.g.
```
my code
```